ti/instrumentation/java/bazel.go

```diff
@@ -236,3 +252,17 @@ func (b *bazelRunner) GetCmd(ctx context.Context, tests []ti.RunnableTest, userA
 	}
 	return fmt.Sprintf("%s %s %s %s", bazelCmd, userArgs, instrArg, testList), nil
 }
+
+// parse module name from rule
+func getModuleFromRule(rule string) string {
+	// parse the rule to extract module
+	splitRule := strings.Split(strings.TrimPrefix(rule, "//"), ":")
+	if strings.Contains(splitRule[0], "/") {
+		splitModule := strings.Split(splitRule[0], "/")
```

Comment:
Ditto.

Reply:
Add Unit Tests for this method to ensure unexpected formats are not going to cause a panic.